.pre-commit-config.yaml

```diff
@@ -130,31 +130,31 @@ repos:
           - --comment-style
           - "..|  |"
   - repo: https://github.com/fsfe/reuse-tool
-    rev: v2.1.0
+    rev: v3.0.1
     hooks:
       - id: reuse
   - repo: https://github.com/pre-commit/mirrors-eslint
-    rev: v8.51.0
+    rev: v8.56.0
     hooks:
       - id: eslint
         additional_dependencies:
-          - "eslint@^8.55.0"
-          - "@angular-eslint/eslint-plugin@17.1.1"
-          - "@angular-eslint/eslint-plugin-template@17.1.1"
-          - "@angular-eslint/template-parser@17.1.1"
+          - "eslint@^8.56.0"
+          - "@angular-eslint/eslint-plugin@17.2.1"
+          - "@angular-eslint/eslint-plugin-template@17.2.1"
+          - "@angular-eslint/template-parser@17.2.1"
           - "eslint-config-prettier@^9.1.0"
-          - "eslint-plugin-import@^2.29.0"
-          - "@typescript-eslint/eslint-plugin@^6.12.0"
-          - "@typescript-eslint/parser@^6.12.0"
+          - "eslint-plugin-import@^2.29.1"
+          - "@typescript-eslint/eslint-plugin@^6.19.0"
+          - "@typescript-eslint/parser@^6.19.0"
           - "eslint-plugin-unused-imports@^3.0.0"
           - "eslint-plugin-deprecation@^2.0.0"
-          - "eslint-plugin-tailwindcss@^3.13.0"
-        args: ["--fix"]
+          - "eslint-plugin-tailwindcss@3.13.1"
```

Comment:
The versions of the additional dependencies should always match the versions of the package.json. Otherwise, eslint will behave differently between running via pre-commit and running via the CLI directly.

Reply:
How do we want to continue with that as here https://github.com/DSD-DBS/capella-collab-manager/pull/1259/files those dependencies are also updated accordingly in the package.json

Reply:
I've merged #1259 and it should no longer be necessary to update those dependencies in this PR.